src/Couchbase.Lite.Shared/API/Database/DatabaseConfiguration.cs

```diff
@@ -76,18 +82,29 @@ public bool FullSync
         }
 
         /// <summary>
-        /// Experiment API. Enable version vector.
+        /// Enables or disables memory-mapped I/O. By default, memory-mapped 
```

Comment:
I worded this differently, based on Jens' comments:
" Advises Core to enable or disable memory-mapped Database files, if possible."

Reply:
Hmmm maybe, but it's also marked as unsupported on the two OS that it doesn't function on, which will give a compiler warning anyway.